lib/index.js

```diff
@@ -183,6 +183,8 @@ internals.finish = function (response, req, onEnd) {
 
     return function () {
 
+        const trailers = {};
```

Comment:
This is odd way of doing it. Why not assign `{}` to `res.trailers` and then just assign directly to that?